### DISCUSSION TITLE
Docker compose v2

### DISCUSSION THREAD

**Discussion by Trapulo** [postalserver/postal#2048] [Help with using Postal]
I noticed that the setup script will ask for docker compose v1 using the docker-compose command.
V1 is dead, I need to install on a Debian 11 that runs Compose v2 (and the CLI is docker compose not docker-compose)

How can this fixed?

thanks

**Comment by willpower232** [comment]
Docker Compose v2 can still be executed directly like V1, if you put it on your shell PATH that is.

That said, it is probably worth updating the install repo to deal with it both ways at some point.

**Comment by Trapulo** [comment]
I changed in the script the only reference to "docker-compose" as "docker compose" and I was able to run the full setup without any problem.

I'm a newbie with docker compose, but I suggest that simple change in the setup script.
